pandas/tests/tools/test_to_datetime.py

```diff
@@ -3500,6 +3482,15 @@ def test_unknown_tz_raises():
         to_datetime([dtstr])
 
 
+def test_unformatted_input_raises():
+    valid, invalid = "2024-01-01", "N"
+    S = Series([valid] * 45000 + [invalid])
```

Comment:
```suggestion
    ser = Series([valid] * 2 + [invalid])
```

Reply:
Agreed. Missed the start_caching_at var in the code. Thanks for this suggestion!